tests/sentry/models/test_outbox.py

```diff
@@ -649,3 +649,64 @@ def test_bulk_operations(self):
 
         assert RegionOutbox.objects.count() == 0
         assert OrganizationMemberTeamReplica.objects.count() == 1
+
+
+class OutboxAggregationTest(TestCase):
+    def setUp(self):
+        shard_counts = {1: (456, "eu"), 2: (789, "us"), 3: (123, "us")}
```

Comment:
Generating 1300 records to do sums could make this test slow. If it does end up being slow we should reduce the number of records generated.

Reply:
Decided just to fix this preemptively, thanks for calling this out.